packages/utilities/src/memoize.test.ts

```diff
@@ -98,6 +98,101 @@ describe('memoizeFunction', () => {
     expect(func('a')).toEqual('a4');
     expect(func('a')).toEqual('a4');
   });
+
+  it('updates the cache if the cached value is null', () => {
+    let returnFalseyValue = true;
```

Comment:
It would be clearer to call this `returnNull`.

Reply:
Done